system-tests/projects/module-api/package.json

```diff
@@ -5,7 +5,7 @@
   "private": true,
   "main": "index.js",
   "scripts": {
-    "test": "mocha --timeout 30000 test/spec.js"
+    "test": "mocha --timeout 45000 test/spec.js"
```

Comment:
Are we doing this because even before the electron bump, we were pretty close to the timeout?

Reply:
Yep, I tried to call this out in the PR description. This test has flaked a few times prior to the recent breaking changes, even in develop. 30s was just a little too tight to verify cypress and run the two specs within one mocha test. 

Please pull this branch down and run verify locally to sanity check the rest of the changes.

Reply:
I tried, could not reproduce this at all. I am on linux though, not an M1mac.

I also tried on windows but I can't even get the binary to build on it (gave up).